kernel/shared/src/main/scala/cats/effect/kernel/Async.scala

```diff
@@ -50,6 +50,14 @@ trait Async[F[_]] extends AsyncPlatform[F] with Sync[F] with Temporal[F] {
   // evalOn(executionContext, ec) <-> pure(ec)
   def evalOn[A](fa: F[A], ec: ExecutionContext): F[A]
 
+  def startOn[A](fa: F[A], ec: ExecutionContext): F[Fiber[F, Throwable, A]] =
+    evalOn(start(fa), ec)
+
+  def backgroundOn[A](
+      fa: F[A],
+      ec: ExecutionContext): Resource[F, F[Outcome[F, Throwable, A]]] =
+    background(fa).map(f => evalOn(f, ec))
```

Comment:
Same problem as above!

```suggestion
    background(evalOn(f, ec))
```

Reply:
Correction, I think this is a more accurate suggestion:

```suggestion
    Resource.make(evalOn(start, ec))(_.cancel).map(_.join)
```

Basically I don't think there's a way to reuse the implementation of `background` itself because of the inversion problem.